app/models/apartment.rb

```diff
@@ -39,4 +39,49 @@ def get_tags
     end
     return hash
   end
+
+
+  #pricing data
+  def get_average_price
+    reviews = Review.where(:apartment_id => self.id)
+    if reviews.size == 0
+      return 0.0
+    end
+
+    sum = 0
+    reviews.each do |review|
+      sum = sum + review.price
+    end
+    return sum/reviews.size
+  end
+
+  def get_min_price
+    reviews = Review.where(:apartment_id => self.id)
```

Comment:
Isn't there a min query function?

Reply:
There are queries that have this functionality.  Here is a link that talks about them.
https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html

Reply:
i'll touch that up in a later submission

Reply:
Just put in a change request.  Once I know the changes Julia mentioned have been properly addressed, I will approve the request.